### ISSUE TITLE
Retroarch Dolphine core apple tv problem

### ISSUE THREAD

**Issue by Anubit3** [libretro/RetroArch#16066]
Hi all
I have problem to load Dolphine core (wii and gamecube) to Retroarch. 
I am using Apple TV latest. 
I tried to use content downloader, manual upload to system Sys in zip and unziped files. 
nothing allow me to see wii and gamecube cores. Even I see uploaded files via Retroarch browser. 
is there any other possibility?
Thanks

**Comment by warmenhoven** [comment]
No, the dolphin core isn't currently available for any arm64 platform, including Apple TV.

Also, for Apple TV, the cores have to be in the IPA before you install it; they can't be downloaded or dynamically added after it's installed, because of Apple's signing rules.

**Comment by RobLoach** [comment]
Make an issue in the core issue queue
